### DISCUSSION TITLE
flash crazyflie

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#1240] [Q&A]
Hi, I have some confuse in flash. so turn of the crazyflie is that means no battery and press the power button ? 
![螢幕擷取畫面 2024-03-20 174255](https://github.com/bitcraze/discussions/assets/118436188/ef7777cb-0191-47d5-b936-0fdddee764d8)

**Comment by ataffanel** [comment]
It means pressing the on/off button to switch off the Crazyflie, the battery needs to be connected.

**Reply by CalvinKoung** [reply]
ok, got it. Thanks
